### PR TITLE
Added devmob logo and start of the login/register

### PR DESCRIPTION
**Pull request by okapifan**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -11,6 +11,10 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+//Route::get('/', function () {
```
Comment (TijmenWierenga) [inline routes/web.php]: You might want to remove this commented out code. This helps you to keep your code clean and tidy.
If you need to comment out code, always explain why it is commented. That really helps other developers understanding why you did it :)

File: resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#app-navbar-collapse">
+                        <span class="sr-only">Toggle Navigation</span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
+
+                    <!-- Branding Image -->
+                    <a href="{{ url('/') }}">
+                        <img style="width: 40%;" alt="logo" src="https://www.google.com/a/devmob.com/images/logo.gif?alpha=1&service=google_default">
```
Comment (TijmenWierenga) [inline resources/views/layouts/app.blade.php]: When you add a logo to your repository, it's best to add that folder to your assets directory. If for some reason this image might get removed/replaced, your website design might break. Add the image somewhere in your `public` directory and then use it like this:

``` blade
<img style="width: 40%;" alt="logo" src="{{ asset('img/logo/devmob.png') }}">
```

File: resources/views/layouts/app.blade.php [hunk 3]
```diff
@@ -29,8 +29,7 @@
 
                     <!-- Branding Image -->
                     <a href="{{ url('/') }}">
-                        <img style="width: 40%;" alt="logo" src="https://www.google.com/a/devmob.com/images/logo.gif?alpha=1&service=google_default">
-                        <!-- {{ config('app.name', 'Laravel') }} -->
+                        <img style="width: 40%;" alt="logo" src="{{ asset('img/logo.png') }}">
```
Comment (TijmenWierenga) [inline resources/views/layouts/app.blade.php]: Last but not least I would move the CSS to a dedicated style guide. Maybe ask @mbicknese to help you organise your stylesheets tomorrow.

File: app/Http/Controllers/Auth/LoginController.php [hunk 4]
```diff
@@ -36,4 +37,21 @@ public function __construct()
     {
         $this->middleware('guest')->except('logout');
     }
+
+    /**
+     * Log the user out of the application.
+     *
+     * @param  \Illuminate\Http\Request  $request
```
Comment (mbicknese) [inline app/Http/Controllers/Auth/LoginController.php]: Personally I like to simplify classnames in the code by using them at the top of the file (as you did with this Request class). You can now simply refer to this class with 'Request' (no namespace needed).

File: public/css/standard.css [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.logo {
+    width: 40%;
+}
```
Comment (mbicknese) [inline public/css/standard.css]: You can set PhpStorm (most editors btw) to auto add a newline at the end of all files. You can look into the reason why, but it comes down to it being a widely accepted standard.

**Review by mbicknese — APPROVED** [review]

**Review by TijmenWierenga — APPROVED** [review]

File: app/Http/Controllers/Auth/LoginController.php [hunk 6]
```diff
@@ -43,7 +43,7 @@ public function __construct()
      * @param  Request $request
      * @return Response
      */
-    public function logout(Request $request): Response
```
Comment (mbicknese) [inline app/Http/Controllers/Auth/LoginController.php]: What error did you get? It should work, so getting an error is weird behaviour.

Comment (mbicknese) [reply app/Http/Controllers/Auth/LoginController.php]: Seeing the returned object is a `RedirectResponse` which does not extend the `Illuminate\Http\Response` the hinted return type is invalid. Which means the docBlock is also wrong.

I'd suggest we go for either hinting it with:

* `Illuminate\Http\RedirectResponse`, verbose and clear, but limiting
* `Symfony\Component\HttpFoundation\Response`, less verbose but we can return any type of response

Both are fine, pick what you think suits the situation best.

Comment (okapifan) [reply app/Http/Controllers/Auth/LoginController.php]: I think I also need to change :Reponse to :RedirectRespose. is that right?

Comment (okapifan) [reply app/Http/Controllers/Auth/LoginController.php]: and also that one above it from `@return Response` to `@return RedirectReponse`

Comment (mbicknese) [reply app/Http/Controllers/Auth/LoginController.php]: si

**Pull request merged**